adaptors/library/jobs/satusehat-create-observations.js

```diff
@@ -0,0 +1,140 @@
+// Create an observation in Satusehat
+
+fn(state => {
```

Comment:
I'd love to see this implemented as an `each` rather than an `fn`. But  I think actually you'd want this to be a `map()` right? Take one array and map the contents into a new one? So let's not change anything but I've made a mental note of this use-case